src/main/java/model/structures/Node.java

```diff
@@ -0,0 +1,119 @@
+package model.structures;
+
+import java.util.Objects;
+
+public class Node {
+    private int residentsNumber;
+    private int illnessCases;
+    private int healthyNumber;
+    private boolean isVaccinated;
+    private int infectingParameter;
+    private int id;
+    private int visitIteration;
+
+    public Node(int id, int residentsNumber, int infectingParameter) {
+        this.residentsNumber = residentsNumber;
+        this.illnessCases = 0;
+        this.healthyNumber = residentsNumber - illnessCases;
+        this.infectingParameter = infectingParameter;
+        this.isVaccinated = false;
+        updateIllnessCases(0);
```

Comment:
Po co to jest wolane w zerowym kroku?

Reply:
Żeby ustawić początkową liczbę chorych

Reply:
a czy illnessCases to nie jest właśnie początkowa liczba chorych?

Reply:
nie, illnessCases -> liczba przypadków zachorowań, wyrażana jest wzorem, który jest w updateIllnessCases oraz w dokumentacji naszego projektu. Zauważ proszę, że żaden z konstruktorów nie ma parametru pozwalającego ustawić wartość tego pola

Reply:
co do mylącego settera to na tym etapie wygenerowałem wszystkie potencjalnie użyteczne settery, jak będziemy wiedzieć, co i jak to się je wyrzuci